app/views/data_quality_reports/version_four/project/README.md

```diff
@@ -0,0 +1,206 @@
+# Project Data Quality Report Calculations
+
+## Overview
+
+### Enrolled Clients
+
+The number of enrollments for the project during the reporting period.
+
+### Enrolled Households
+
+The number of enrollments with `RelationshipToHoH` blank or '1' for the project during the reporting period.
+
+### Active Clients
+
+The number of enrollments for the project during the reporting period for entry/exit projects, or with service during the reporting period for night-by-night projects.
+
+### Active Households
+
+The number of enrollments with `RelationshipToHoH` blank or '1' for the project during the reporting period for entry/exit projects, or with service during the reporting period for night-by-night projects.
+
+### Entering Clients
+
+The number of enrollments for the project during the reporting period with an `EntryDate` during the reporting period.
+
+### Entering Households
+
+The number of enrollments for the project during the reporting period with `RelationshipToHoH` blank or '1', and with an exit record and the `EntryDate` during the reporting period.
```

Comment:
Is this correct that it has a requirement to have an exit record?

Reply:
No -- copy/paste error.